tests/saw/spec/handshake/rfc_handshake_tls12.cry

```diff
@@ -69,6 +74,7 @@ type MessageType = [4]
 (finished           : MessageType) = 9
 (newSessionTicket   : MessageType) = 10
 (certificateStatus  : MessageType) = 11
+(encryptedExtensions : MessageType) = 12
 
```

Comment:
Because we're not doing draft 4, the name of this message isn't right. Draft 3 just calls it NextProtocol.

This applies to some other places in this proof.

Reply:
Done.